hawc/apps/epiv2/serializers.py

```diff
@@ -1,77 +1,168 @@
 from django.db import transaction
 from rest_framework import serializers
 
-from ..common.serializers import FlexibleChoiceField, IdLookupMixin
+from ..assessment.models import DSSTox
+from ..assessment.serializers import DSSToxSerializer
+from ..common.serializers import ArrayedFlexibleChoiceField, FlexibleChoiceField, IdLookupMixin
 from ..epi.serializers import StudyPopulationCountrySerializer
+from ..study.models import Study
 from ..study.serializers import StudySerializer
-from . import constants, models
+from . import constants, mixins, models
 
 
 class ChemicalSerializer(serializers.ModelSerializer):
+    dsstox_id = serializers.PrimaryKeyRelatedField(
+        write_only=True,
+        source="dsstox",
+        queryset=DSSTox.objects.all(),
+        required=False,
+        allow_null=True,
+    )
+    dsstox = DSSToxSerializer(read_only=True)
+
     class Meta:
         model = models.Chemical
-        fields = ["id", "name", "dsstox"]
+        fields = ["id", "design", "name", "dsstox_id", "dsstox"]
 
 
 class ExposureSerializer(serializers.ModelSerializer):
     exposure_route = FlexibleChoiceField(choices=constants.ExposureRoute.choices)
+    biomonitoring_matrix = FlexibleChoiceField(
+        choices=constants.BiomonitoringMatrix.choices, allow_blank=True
+    )
+    biomonitoring_source = FlexibleChoiceField(
+        choices=constants.BiomonitoringSource.choices, allow_blank=True
+    )
 
     class Meta:
         model = models.Exposure
-        exclude = ["design", "created", "last_updated"]
-
+        exclude = ["created", "last_updated"]
+
+
+class ExposureLevelSerializer(mixins.BelongsToSameDesignMixin, serializers.ModelSerializer):
+    variance_type = FlexibleChoiceField(choices=constants.VarianceType.choices)
+    ci_type = FlexibleChoiceField(choices=constants.ConfidenceIntervalType.choices)
+
+    belongs_to_same_design_fields = [
```

Comment:
move the `belongs_to_same_design_fields` down below all the other serializer fields, and should we rename this to `same_design_fields`. Do this for other serializer modes as well